scripts/qa_crowbarsetup.sh

```diff
@@ -2040,6 +2042,22 @@ function onadmin_post_allocate()
             ssh $node "getent passwd glance >/dev/null || \
                 useradd -r -g glance -u 450 -d /var/lib/glance -s /sbin/nologin -c \"OpenStack glance Daemon\" glance"
         done
+
+        if iscloudver 6plus && [[ $want_sbd = 1 ]] ; then
+            zypper --gpg-auto-import-keys -p http://download.opensuse.org/repositories/devel:/languages:/python/SLE_12_SP1/ --non-interactive install python-sh
+            wget --no-check-certificate https://raw.githubusercontent.com/SUSE-Cloud/automation/master/scripts/iscsictl.py
```

Comment:
Shouldn't the mkcloud script scp the script (since we have it in the git clone already)?